### PR TITLE
Render group icon in ChatList

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/deltachat/deltachat-desktop/issues/372

### CODE REVIEW

**Comment by karissa** [comment]
nice @ralphtheninja !

For code organization purposes, wondering if we can have a `Chat.js` module or somehow incorporate it into the same file as `ChatView.js` -- thinking similar to what we had/have for Message, which contains some helper functions in there: and exposes `render` (for creating the React element) and `convert` for transferring the delta-api to api needed for renderer..

**Comment by ralphtheninja** [comment]
> For code organization purposes, wondering if we can have a `Chat.js` module or somehow incorporate it into the same file as `ChatView.js` -- thinking similar to what we had/have for Message, which contains some helper functions in there: and exposes `render` (for creating the React element) and `convert` for transferring the delta-api to api needed for renderer..

I trust your judgement :)

**Comment by karissa** [comment]
Yeah, this way we can just pass `selectedChat` and there is a `isGroup` property on that, rather than passing `isGroup` as a property of the ChatView, or as an extra call/require in the Menu file.

**Comment by karissa** [comment]
(similar to `chat.isVerified`)

**Comment by ralphtheninja** [comment]
> (similar to `chat.isVerified`)

Aah think I know what you mean. I'll tweak it a bit.

File: src/renderer/components/Menu.js [hunk 1]
```diff
@@ -77,7 +76,7 @@ class DeltaMenu extends React.Component {
 
     const tx = window.translate
 
-    const isGroup = isGroupChat(selectedChat)
+    const isGroup = selectedChat.isGroup
```
Comment (ralphtheninja) [inline src/renderer/components/Menu.js]: @karissa Something like this?

Comment (karissa) [reply src/renderer/components/Menu.js]: yeah better!

**Comment by ralphtheninja** [comment]
@karissa Good to merge?

**Pull request merged**